### DISCUSSION TITLE
可否使在进入视频页面时自动暂停?

### DISCUSSION THREAD

**Discussion by lolisix** [the1812/Bilibili-Evolved#1926] [功能建议 / Ideas]
每次打开视频网页,视频都会自动播放,这对我这种一口气打开多个网页,再慢慢观看的人来说不太友好.

**Comment by Coulomb-G** [comment]
自动播放b站自带有开关,你关了就自动暂停了

**Reply by lolisix** [reply]
我的天呀,我之前记得是有开关,奈何找了半天都没找到,以为记错了,没想到藏在了视频播放窗口里面.
我最开始本能地在个人设置相关页面找,实在是太蠢了,太丢人了.(=σ′д｀)σ
